js/common/charts.d.ts

```diff
@@ -260,6 +260,12 @@ export type VisualRange = {
     startValue?: number | Date | string;
 };
 
+/**
+ * @public
+ * @namespace DevExpress.common.charts
+ */
+export type ValueAxisVisualRangeUpdateMode = 'auto' | 'keep' | 'reset';
```

Comment:
We already export enum with same name/values from polar_chart.d.ts:
https://github.com/DevExpress/DevExtreme/blob/23_1/js/viz/polar_chart.d.ts#L99
Consider exporting/using single enum/type, may be from common/charts.d.ts